### PR TITLE
added tests for blocking-queue task

### PR DESCRIPTION
**Pull request by KseniiaMakarova**

### CODE REVIEW

**Review by Max-Pochepets — APPROVED** [review]

File: src/test/java/core/basesyntax/BlockingQueueTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        producer.start();
+        Thread.sleep(500);
+        Assert.assertNotEquals("Your `put()` method should not put the producer thread "
+                + "in the WAITING state if there is still space in the queue",
+                Thread.State.WAITING, producer.getState());
+    }
+
+    @Test
+    public void put_fullCapacity_ok() throws InterruptedException {
+        blockingQueue = new BlockingQueue<>(49);
```
Comment (boroda4436) [inline src/test/java/core/basesyntax/BlockingQueueTest.java:25]: Maybe it's better to call class MyBlockingQueue and avoid confusion with [interface](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/BlockingQueue.html)?
image

File: src/test/java/core/basesyntax/BlockingQueueTest.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        consumer.start();
+        Thread.sleep(100);
+        Assert.assertNotEquals("Your producer thread should continue working "
+                + "if some space appeared in the queue",
+                Thread.State.WAITING, producer.getState());
+    }
+
+    @Test
+    public void take_ok() throws InterruptedException {
+        blockingQueue = new BlockingQueue<>(50);
```
Comment (boroda4436) [inline src/test/java/core/basesyntax/BlockingQueueTest.java:43]: Is the capacity parameter required? We haven't test cases to verify the behavior that happens if capacity is reached.
```suggestion
        blockingQueue = new BlockingQueue<>();
```

Comment (KseniiaMakarova) [reply src/test/java/core/basesyntax/BlockingQueueTest.java:43]: discussed, thanks

File: src/test/java/core/basesyntax/BlockingQueueTest.java (hunk 1 above)
Comment (KseniiaMakarova) [reply src/test/java/core/basesyntax/BlockingQueueTest.java:25]: discussed

**Pull request merged**